ReSharper.FSharp/test/src/FSharp.Intentions.Tests/src/QuickFixes/AddMissingSeqFixTest.fs

```diff
@@ -0,0 +1,20 @@
+namespace JetBrains.ReSharper.Plugins.FSharp.Tests.Intentions.QuickFixes
+
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.QuickFixes
+open JetBrains.ReSharper.Plugins.FSharp.Tests
+open NUnit.Framework
+
+[<FSharpTest>]
```

Comment:
@edgarfgp Could you try using `AssertCorrectTreeStructure` attribute here? It should make modifying the tree easier.

Reply:
Sure.